### PR TITLE
[TW-1361] Public API Network methods for publishing

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
Documented different methods API publishers can use to publish APIs to the Public Network.

[Beta site](https://tw-1361-public-network-methods-for-publishers.learning.postman-beta.com/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace/)

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! Awesome new page 🌟

File: src/pages/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+When you're ready to share your collections with your API's consumers, you can use code to create a `GET` request that gets the collection you want to publish, then you can create a `PUT` request that publishes the collection. Use the `/collections/collection-id` endpoint to get and publish your collections.
+
+> You can also create and send the requests in a [collection](/docs/collections/using-collections/) or [Flow](/docs/postman-flows/gs/flows-overview/).
+
+To share a collection to a public workspace using the Postman API, do the following:
+
+1. [Create a collection](/docs/collections/using-collections/#creating-collections) (source collection) in a non-public workspace, and create another collection (destination collection) in a public workspace that's available on the Public API Network.
+
+    Then get the collection IDs for both collections. Select the information icon <img alt="Information icon" src="https://assets.postman.com/postman-docs/icon-information-v9-5.jpg#icon" width="16px"> in the right sidebar, and copy the collection IDs for later use.
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace.md:38]:
```suggestion
1. Get the collection IDs for both collections. Select the information icon <img alt="Information icon" src="https://assets.postman.com/postman-docs/icon-information-v9-5.jpg#icon" width="16px"> in the right sidebar, and copy the collection IDs.
```

File: src/pages/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+1. [Get a Postman API key](/docs/developer/postman-api/authentication/#generate-a-postman-api-key) in the team where your source collection is stored. If the destination collection is in a different team, you must also get a Postman API key in that team.
+1. Create a `GET` request to a collection in a non-public workspace that you want to publish. The request returns a collection object in the response body. The object contains details about the source collection you can use to publish it.
+
+    Use the following URL in your request:
```
Comment (jkonrath-postman) [inline src/pages/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace.md:43]: Minor nit, but should we tell people to use cURL here, or just document how to do it in Postman?

**Review by jkonrath-postman — APPROVED** [review]
I added a few suggestions, but LGTM. Make sure to pull in develop and work out the leftnav conflict.

File: src/pages/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace.md (hunk 2 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace.md:43]: I see what you're saying. I'll follow up with Kyle on this.

Comment (MatthewPinsker-postman) [reply src/pages/docs/collaborating-in-postman/public-api-network/maintain-api-in-workspace.md:43]: Discussed with @jkonrath-postman, and we'll leave the content as is for now.

**Review by stcalica — APPROVED** [review]
LGTM!

**Pull request merged**